docs/pbc/all/payment-service-providers/unzer/howto-tips-use-cases/understand-payment-method-in-checkout-process.md

```diff
@@ -55,18 +63,18 @@ All payment methods enabled for a marketplace business model in Unzer are displa
 
 #### Example
 
-In your Unzer account, the following payment methods are enabled for the marketplace business model
+In your Unzer account, the following payment methods are enabled for the marketplace business model.
 
 * Credit Card
 * Unzer Bank Transfer
```

Comment:
```suggestion
* Marketplace Credit Card
* Marketplace Bank Transfer
```

Reply:
No @michbeck this is internal behavior of our integration we must not put those details in the doc, for the business Credit Card and Marketplace credit card are the same